scss/responsive/_xsmall.scss

```diff
@@ -1,4 +1,5 @@
 .oo-xsmall {
+  height: 100%;
```

Comment:
Does this have any effect on the regular player?

Reply:
I haven't noticed anything with the regular player.